src/components/Clock.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import '../App.scss';
+import './Clock.scss';
+
+interface Props {
+  clockName: string,
+}
+
+type State = {
+  time: Date,
+};
+
+class Clock extends React.Component<Props, State> {
+  state = {
+    time: new Date(),
+  };
+
+  timer = setInterval(() => {
+    this.state.time.toLocaleTimeString();
```

Comment:
That's unnecesary operation, you're not saving it's result
```suggestion
```

Reply:
fixed